### PR TITLE
Updated webpack-dev-server version

### LINKED ISSUE #542: Webpack-dev-server should be updated
It should be updated to a newer version.

### PR DESCRIPTION
**Pull request by markus659**
Fixes #542.

### CODE REVIEW

**Comment by tonnyandersson** [comment]
Tested and build went through just fine. But I get _ReferenceError: Origo is not defined_ when trying to load the build. Works fine when changing script source to origo.min.js, though,

**Comment by jokd** [comment]
I think that’s the expected behaviour. It’s a bug, but an old one.

**Comment by tonnyandersson** [comment]
@jokd Yeah, I know about that bug. Was hoping this PR would deal with that somehow.

File: tasks/webpack.dev.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ const common = require('./webpack.common.js');
 
 module.exports = merge(common, {
   output: {
-    path: `${__dirname}/../js`,
```
Comment (daRasmussen) [inline tasks/webpack.dev.js]: Removing path did not work me first I had to change path to: 
```
path: `${__dirname}`,
```
Then webpack-dev-server ran just fine. Even if I removed path altogether.

File: package.json [hunk 2]
```diff
@@ -54,7 +54,7 @@
     "webpack": "^4.27.1",
     "webpack-bundle-analyzer": "^3.0.3",
     "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "3.1.8",
+    "webpack-dev-server": "3.1.14",
```
Comment (daRasmussen) [inline package.json]: npm audit suggests: 
```
webpack-dev-server@3.2.1
"webpack-dev-server": "^3.2.1",
```

**Comment by jokd** [comment]
I think we can step up to 3.2.1 as well. Tested and works well (both with and without path)
I had to manually install acorn though, not sure why

**Comment by markus659** [comment]
Does it look OK now @daRasmussen @jokd?

**Pull request merged**